src/main/java/racingcar/RacingCar.java

```diff
@@ -0,0 +1,29 @@
+package racingcar;
+
+public class RacingCar {
+
+    public void nameCheck(String carName) {
```

Comment:
```suggestion
    public void checkName(String carName) {
```
메서드 네이밍 규칙(by. 구글 자바 컨벤션)에 따르면 메서드 네이밍은 동사, 동사구에 lowerCamelCase 😉 (앞에 유사한 피드백이 있어서 언급만 하고 넘어갈께요!   이 부분은 전체적으로 확인해서 반영해주세요 😉👏)